src/test/java/task40/DownloadPercentageTest.java

```diff
@@ -0,0 +1,38 @@
+package task40;
+
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Wait;
+import task40.helpes.BaseTest;
+import task40.helpes.DownloadPercentagePage;
+
+import java.time.Duration;
+
+import static task40.helpes.TestConstants.*;
+
+public class DownloadPercentageTest extends BaseTest {
+
+    public DownloadPercentageTest() {
+        super(DOWNLOAD_PERCENTAGE_URL);
+    }
+
+    @Test
+    public void downloadPercentageTest() throws InterruptedException {
+        DownloadPercentagePage page = new DownloadPercentagePage(super.driver);
+        WebElement button = page.getButtonWebElement();
+
+        button.click();
```

Comment:
Your test shouldn't work with webElements. If you need to click , create method that will do it in page class.

Reply:
Done